js/index.js

```diff
@@ -1,41 +1,59 @@
-function createBookElement(title, author, id) {
-  return `
-    <div id="${id}">
-      <p>Title: ${title}<br>Author: ${author}</p>
-      <button type="button" onclick="removeBook('${id}')">Remove</button>
-      <hr/>
-    </div>`;
+/*  eslint-disable max-classes-per-file, no-unused-vars, class-methods-use-this */
```

Comment:
- [ ] Please do not disable linters in your project as they help enforce good coding standards

Reply:
@MarcoDiaz2000 , let me double check and revert

Reply:
Thanks

Reply:
@wayungi Thanks

Reply:
hello @MarcoDiaz2000 , In my opinion, the exception to disable linters allows for only 2 rules as seen below.
![rules](https://user-images.githubusercontent.com/99656687/234540963-33e7876c-f91c-4e10-88dc-01f5de1e5c8b.png)
-  [ ] You have however disabled 3 linter rules. I suggest that you only disable the exception provided as this will help you learn to write better quality code

Reply:
Ok, @wayungi I will follow your recommendation and only the two suggested rules will be disabled.

When I make the change, do I tell you about it here or do I ask for a new revision?

Reply:
Please @MarcoDiaz2000  request a review as you have always done, another code reviewer will take up and review the project.

Reply:
Ok @wayungi Thanks